tests/functional/codegen/calling_convention/test_external_contract_calls.py

```diff
@@ -2503,9 +2503,9 @@ def foo(a: {typ}):
     pass
     """
     c1 = get_contract(code)
-    sig = keccak(f"foo({typ})".encode()).hex()[:10]
+    sig = keccak(f"foo({typ})".encode()).hex()[:8]
```

Comment:
i think clearer would be: `keccak(...)[:4].hex()`

or even just using `vyper.utils.method_id()`